tests/test_germline_dna.yml

```diff
@@ -49,5 +49,5 @@
     - path: "test-output/multisample_variants/multisample.vcf.gz.tbi"
     - path: "test-output/multisample_variants/stats"
     - path: "test-output/samples/NA12878/lib_U0a/NA12878-U0a.markdup.bam"
-    - path: "test-output/samples/NA12878/lib_U0a/NA12878-U0a.markdup.bai"
-    - path: "test-output/samples/NA12878/lib_U0a/metrics/NA12878-U0a.markdup.flagstats"
\ No newline at end of file
+    - path: "test-output/samples/NA12878/lib_U0a/metrics/NA12878-U0a.markdup.flagstats"
+    - path: "test-output/samples/NA12878/lib_U0a/metrics/wgs2-paired-end-lib1.markdup.flagstats"
```

Comment:
This doesn't seem to be correct, the sample and library names earlier in this path don't match the ones in the file name. Actually, it looks like this line shouldn't be here at all.

Reply:
Merge error.  Removed it.